Excercise #?

numpy-ai opened the pull request:
yeah

numpy-ai committed:
Combine var Excercise

yeah

numpy-ai committed:
Excercise #?

mintjd123 commented:
Combine Var Result
> var1
[1] 1 2 3 4 5
> var2
[1] 1 2 3 4 5
> var3
[1] 1 2 3 4 5
> var4
[1] 1 3 5 7 9
> var5
[1]  1  4  7 10
> var1 + var2
[1]  2  4  6  8 10

mintjd123 commented:
String EX result
> str1 <- "a"
> str1
[1] "a"
> str1 <- "a"
> str1
[1] "a"
> str2 <- "text"
> str2
[1] "text"
> str3 <- "Hello World!"
> str3
[1] "Hello World!"
> str4 <- c("a", "b", "c")
> str4
[1] "a" "b" "c"
> str5 <- c("Hello!", "World", "is", "good!")
> str5
[1] "Hello!" "World"  "is"     "good!" 
> str1 + 2
Error in str1 + 2 : 이항연산자에 수치가 아닌 인수입니다
문자열에 2를 더했더니 안 된다. 문자열로 바뀌는 게 아니라 그냥 수치로 인식해서 연산이 안 되는 모양이다.

numpy-ai committed:
Excercise #?

Easily string Function.R result
> x <- c(1, 2, 3)
> x
[1] 1 2 3
> mean(x)
[1] 2
> max(x)
[1] 3
> min(x)
[1] 1
> paste(str5, collapse = ",") #","을 구분자로 문자를 합침
[1] "Hello!,World,is,good!"
> paste(str5, collapse = " ") #collapse같은 함수의 옵션을 설정하는 명령어를 파라미터라고 한다.
[1] "Hello! World is good!"
> x_mean <- mean(x)
> x_mean
[1] 2
> str5_paste <- paste(str5, collapse = " ")
> str5_paste
[1] "Hello! World is good!"

numpy-ai committed:
Create ggplot2 Exercise (packages).R

numpy-ai committed:
Create Quiz1.R

1. 다섯 명의 학생이 시험을 봤다. 학생들의 시험 점수를 담고 있는 변수를 만들어 출력해라. (점수는 80, 60, 70, 50, 90)
2. 전체 평균을 구해라
3. 전체 평균 변수를 만들고 출력해라

numpy-ai committed:
Data frame Exercise.R

numpy-ai committed:
Quiz2.R

제품에 사과, 딸기, 수박, 그에 맞는 가격과 판매량이 들어가 있는 데이터프레임을 만들고 가격과 판매량의 전체 평균을 구하라.

결과 값
> fruit_info
  제품 가격 판매량
1 사과 1800     24
2 딸기 1500     38
3 수박 3000     13

> mean(fruit_info$가격)
[1] 2100
> mean(fruit_info$판매량)
[1] 25

numpy-ai committed:
Outer data import(xlsx).R

외부에 있는.xlsx 확장자 엑셀 파일을 불러온다.

numpy-ai committed:
Outer data import(csv).R

외부에 있는.csv 확장자 엑셀 파일을 불러온다.

numpy-ai committed:
Write Dataframe.R

데이터 프레임을 작성해서 저장한다.
csv로 저장 할 수도 있고 RData 파일인 rda로 저장할 수도 있다.
rda로 불러올 때는 새 변수에 할당할 필요가 없다.

numpy-ai committed:
Know about data.R

numpy-ai committed:
Know about mpg

ggplot2에서 기본으로 제공해주는 mpg 데이터를 알아보았다.
